addons/website_event_questions/static/src/js/website_event_questions.js

```diff
@@ -0,0 +1,8 @@
+odoo.define('website_event_questions.website_event_questions', function (require) {
```

Comment:
@tde-banana-odoo reason behind this code is to reuse EventRegistrationForm defined in website_event.js, I want to bind selection elements with select2, note that we convert event registration popup into widget and convert it to new module style.